### DISCUSSION TITLE
New Ryzen 8845hs

### DISCUSSION THREAD

**Discussion by ProFFs** [DavidS95/Smokeless_UMAF#75] [Q&A]
Has anyone tried the utility on honor magicbook x16 plus on ryzen 8845hs or other laptops with ryzen 8845hs, does it work?

**Comment by melodictsk** [comment]
It is not working on bios 1.06 (((((

**Reply by melodictsk** [reply]
works if secure boot is off!!

**Comment by Blaubluter** [comment]
It would be a shame if it didn't work.. Have you already opened the boot menu?


Then


Shut down Windows, press Shift while selecting Shut Down, and then select Select USB.

**Comment by guerillaengineer** [comment]
still works. remember you have to go into your bios settings and save changes from there after
